### DISCUSSION TITLE
QNAP support

### DISCUSSION THREAD

**Discussion by hello-world-dot-c** [garethgeorge/backrest#336] [Show and tell]
User QoolBox has added the BackRest server to the MyQNAP.org repository for QNAP NAS devices:

https://www.myqnap.org/product/backrest/

Something to add to the main readme under [Running on Linux](https://github.com/garethgeorge/backrest#running-on-linux)?

**Comment by garethgeorge** [comment]
Hey, I’m not very familiar with QNAP but I’m pretty hesitant to add non-official packages to the readme. Anything direct-linked should be a part of backrest’s build system and automatically updated when it runs releases. The only exception there is the arch Linux package at the moment.
